models/product/src/generator.ts

```diff
@@ -0,0 +1,31 @@
+import { ClientLogging, Reference } from '@commercetools-test-data/commons';
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import * as ProductCatalogData from '@commercetools-test-data/product-catalog-data'; // TODO: fix import
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { productPriceMode } from './constants';
+import { TProduct } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/products#product
+
+const generator = Generator<TProduct>({
+  fields: {
+    id: fake((f) => f.datatype.uuid()),
+    version: fake((f) => f.datatype.number()),
+    key: fake((f) => f.lorem.slug()),
+    productType: fake(() => Reference.random().typeId('product-type')),
+    masterData: fake(() => ProductCatalogData.random()),
+    taxCategory: fake(() => Reference.random().typeId('tax-category')),
+    state: fake(() => Reference.random().typeId('state')),
+    reviewRatingStatistics: null,
+    priceMode: oneOf(...Object.values(productPriceMode)),
+    skus: [],
```

Comment:
Might be nice to see a stubbed out sku string value here

Reply:
I'm curious what your care amount is here. I'm assuming (and this is a big assumption) that in the GraphQL API that the skus are pulled from the product's variants and exposed via this field. Is that worthy of a transformer field or do we leave it here? My cares are so low.

Reply:
Yeah I agree, good point. Care amount also low, feels like there's not really a dire need to address it since we're not even using this model